makefile

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+default: 
+	@cat makefile
+
+pg17192.txt:
+	bash get_texts.sh
+
+raven_line_count:
+	bash raven_line_count.sh
+
+raven_word_count:
+	bash raven_word_count.sh
+
+raven_counts:
+	bash raven_counts.sh
+
+total_lines:
+	bash total_lines.sh
+
+total_words:
+	bash total_words.sh
+
+env:
+        python3 -m venv env; source env/bin/activate; pip install --upgrade pip;
```

Comment:
this line is showing up without a 'tab', instead it's using spaces.  At least in the version I pull when I got to WEEK5 branch.  Can you please check WEEK5 and fix if it's still a tab?  It won't run on my machine after a git pull
Thanks!